COM-32: Fix padding of DamVideoBlock and YouTubeVideoBlock (#1335)
vivid-planet/comet: .changeset/wise-sloths-remember.md

---
"@comet/blocks-admin": patch
---

> Fix padding behaviour of `YoutubeVideoBlock` and `DamVideoBlock` in paper components

Reviewer: ```suggestion
Fix padding behavior of `YoutubeVideoBlock` and `DamVideoBlock` when used inside `AdminComponentPaper`
```
